### PR TITLE
Update monitoring dashboard documentation with health check

### PR DESCRIPTION
**Pull request by madushadhanushka**
Update docs for MI Dashboard health check

### CODE REVIEW

File: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md [hunk 1]
```diff
@@ -252,6 +252,15 @@ Now you can view details of artifacts, update artifacts, and perform various oth
 
 <a href="{{base_path}}/assets/img/integrate/monitoring-dashboard/dashboard-artifact-list.png"><img src="{{base_path}}/assets/img/integrate/monitoring-dashboard/dashboard-artifact-list.png" width="300"></a>
 
+## Monitoring health of MI Dashboard
```
Comment (Mariangela) [inline en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:255]:
```suggestion
## Monitoring health of MI Dashboard

```

File: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md [hunk 2]
```diff
@@ -252,6 +252,15 @@ Now you can view details of artifacts, update artifacts, and perform various oth
...
+MI Monitoring Dashboard provide a health check endpoint that can be used to monitor the status of the Dashboard. The following API format can be used to invoke the health API:
```
Comment (Mariangela) [inline en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:256]:
```suggestion
MI Monitoring Dashboard provide a health check endpoint that can be used to monitor the status of the Dashboard. The following API format can be used to invoke the health API:

```

File: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md [hunk 3]
```diff
@@ -252,6 +252,15 @@ Now you can view details of artifacts, update artifacts, and perform various oth
...
+```
+https://localhost:9743/dashboard/api/healthz
+```
+If MI Dashboard is up and running properly, Then the following response will be responded:
```
Comment (Mariangela) [inline en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:260]:
```suggestion

If MI Dashboard is up and running properly, the following response will appear.

```

File: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md (hunk 2 above)
Comment (Mariangela) [inline en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:256]:
```suggestion

The MI Monitoring Dashboard provides a health check endpoint that can be used to monitor the status of the Dashboard. The following API format can be used to invoke the health API:

```

Comment (Mariangela) [inline en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:256]:
```suggestion

The MI Monitoring Dashboard provides a health check endpoint that can be used to monitor the status of the Dashboard. The following API format can be used to invoke the health API.

```

File: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md (hunk 3 above)
Comment (Mariangela) [inline en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:260]:
```suggestion

If the MI Dashboard is up and running properly, the following response will appear.

```

File: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md [hunk 4]
```diff
@@ -252,6 +252,15 @@ Now you can view details of artifacts, update artifacts, and perform various oth
 
 <a href="{{base_path}}/assets/img/integrate/monitoring-dashboard/dashboard-artifact-list.png"><img src="{{base_path}}/assets/img/integrate/monitoring-dashboard/dashboard-artifact-list.png" width="300"></a>
 
+## Monitoring health of MI Dashboard
+MI Monitoring Dashboard provide a health check endpoint that can be used to monitor the status of the Dashboard. The following API format can be used to invoke the health API:
+```
+https://localhost:9743/dashboard/api/healthz
```
Comment (Mariangela) [inline en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:258]: @madushadhanushka , is the "z" at the end of the URL correct?

**Review by Mariangela — APPROVED** [review]

File: en/docs/observe/mi-observe/working-with-monitoring-dashboard.md (hunk 4 above)
Comment (madushadhanushka) [reply en/docs/observe/mi-observe/working-with-monitoring-dashboard.md:258]: yes. This link is same as MI health check endpoint URL

**Pull request merged**